test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/hs201t002.cpp

```diff
@@ -665,11 +666,6 @@ jint Agent_Initialize(JavaVM *jvm, char *options, void *reserved) {
 
     NSK_DISPLAY0("Enable events\n");
 
-    enableEvent(jvmti, JVMTI_EVENT_CLASS_LOAD, testedThread);
```

Comment:
This code executed on agent initialization, the test has no chance to set testThread.
Moved to hs201t002.setThread()